about.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script>
+      src="https://kit.fontawesome.com/cb6c5078b1.js"
+      crossorigin="anonymous"
+      </script>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+  <link rel="stylesheet" href="about.css">
+  <title>About Burning Circle Flight Academy</title>
+</head>
+<body>
+  <div class="top-section">
+    <div class="bar-container">
+    <ul class="bars">
+      <li>
+        <i class="fab fa-facebook-f"></i>
+      </li>
+      <li>
+        <i class="fab fa-twitter"></i>
+      </li>
+      <li>
+        <h4>English</h4>
+      </li>
+      <li>
+        <h4>My Page</h4>
+      </li>
+      <li>
+        <h4>Logout</h4>
+      </li>
+    </ul>
+    </div>
+  </div>
```

Comment:
- [ ] This element should be a part of the `header` element, kindly update it on both pages.
- [ ] I would suggest omitting the usage of one of the containers `div`, it's a bit impractical to wrap the same elements in two containers.
- [ ] Social media links are missing from the top section of the `header`, please add them making sure it's identical on both pages.

Reply:
may I call you on zoom

Reply:
done